static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-baseline.js

```diff
@@ -60,6 +67,26 @@ const isEpicWithinViewLimit = (test: EpicABTest): boolean => {
     return withinViewLimit && enoughDaysBetweenViews;
 };
 
+const changeLinks = (cssClass: string, link: string) => {
+    [...document.getElementsByClassName(cssClass)].forEach(el => {
+        el.href = link;
+    });
+};
+
+const changeHeaderLinks = (
+    becomeSupporterLink: string,
+    subscribeLink: string
+) => {
+
+    changeLinks('js-become-member', becomeSupporterLink);
+    changeLinks('js-subscribe', subscribeLink);
+
+    changeLinks('js-become-member-new-header', becomeSupporterLink);
```

Comment:
I feel it is safer to have a new class name, just in case something else is using the `js-become-member` also.